heartbeat/processor/peerAuthenticationRequestsProcessor.go

```diff
@@ -34,7 +34,7 @@ type ArgPeerAuthenticationRequestsProcessor struct {
 	Epoch                   uint32
 	MinPeersThreshold       float32
 	DelayBetweenRequests    time.Duration
-	MaxTimeout              time.Duration
+	MaxTimeoutForRequests   time.Duration
```

Comment:
I've seen that `MaxTimeoutInSec` -> `PeerAuthenticationMaxTimeoutForRequestsInSec`
Wouldn't it respect the same convention if it would be to call it:
```suggestion
	PeerAuthenticationMaxTimeoutForRequests   time.Duration
```
and propagate it to all the other files?

Reply:
not really, as this already happens inside a peer authentication component